sdk/Lusid.Sdk.Tests/tutorials/Instruments/ForwardRateAgreementExamples.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class ForwardRateAgreementExamples: DemoInstrumentBase
+    {
+        /// <inheritdoc />
+        protected override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
+        {
+            // Nothing to upsert specifically for Forward Rate Agreements (for now - ANA-1301).
+        }
+
+        /// <inheritdoc />
+        protected override void GetAndValidatePortfolioCashFlows(LusidInstrument instrument, string scope, string portfolioCode, string recipeCode, string instrumentID)
+        {
+            // ANA-1301
+        }
+
+        [LusidFeature("F5-24")]
+        [Test]
+        public void ForwardRateAgreementCreationAndUpsertionExample()
+        {
+            // CREATE a future instrument (that can then be upserted into LUSID)
+            var forwardRateAgreement = (ForwardRateAgreement) InstrumentExamples.CreateExampleForwardRateAgreement();
+
+            // ASSERT that it was created
+            Assert.That(forwardRateAgreement, Is.Not.Null);
+            
+            // CAN NOW UPSERT TO LUSID
+            var uniqueId = forwardRateAgreement.InstrumentType + Guid.NewGuid().ToString(); 
+            var instrumentsIds = new List<(LusidInstrument, string)>{(forwardRateAgreement, uniqueId)};
+            var definitions = TestDataUtilities.BuildInstrumentUpsertRequest(instrumentsIds);
+            var upsertResponse = _instrumentsApi.UpsertInstruments(definitions);
+            ValidateUpsertInstrumentResponse(upsertResponse);
+
+            // CAN NOW QUERY FROM LUSID
+            GetInstrumentsResponse getResponse = _instrumentsApi.GetInstruments("ClientInternal", new List<string> { uniqueId });
```

Comment:
you should qualify the get with the `asAt` returned from the upsert to avoid any DEI related eventual consistency issues